task16.js

```diff
@@ -0,0 +1,19 @@
+// Write a function that checks if a string is a palindrome.
+
+function isPalindrome(str) {
+  //   ES6
+  const strToCompare = str.toLowerCase();
+  const reversedStr = strToCompare.split("").reverse().join("");
+
+  return strToCompare === reversedStr;
```

Comment:
u can do all ur operations it `return`
`return str.toLowerCase().split("").reverse().join("") === reversedStr;`